source/user-manual/reference/ossec-conf/gcp-bucket.rst

```diff
@@ -107,7 +107,7 @@ For example ``<credentials_file>wodles/gcp-bucket/credentials.json</credentials_
 bucket\\path
 ^^^^^^^^^^^^
 
-If defined, the path or prefix for the bucket.
+Bucket path or prefix.
```

Comment:
Since this is an optional parameter, the conditional is necessary in the description of it.

Reply:
I removed it to improve the consistency of the descriptions, because there are other non-mandatory options that do not have conditional statements. I think that adding the table with that information solves this.